openfpga_flow/regression_test_scripts/basic_reg_test.sh

```diff
@@ -166,6 +166,7 @@ run-task basic_tests/bus_group/full_testbench_implicit_mapping $@
 
 echo -e "Testing fix pins features";
 run-task basic_tests/fix_pins $@
+run-task basic_tests/pin_constrain $@
```

Comment:
Wrong test case name here:
pin_constrain -> constrain_pin_location

Error message from CI: 
https://github.com/lnis-uofu/OpenFPGA/runs/7439283112?check_suite_focus=true#step:7:4765

Reply:
Thanks. Fixed it.